src/Evolution/Initialization/Helpers.hpp

```diff
@@ -16,7 +16,6 @@
 
 /// Items for initializing the DataBox%es of parallel components
 namespace Initialization {
-
```

Comment:
This function can probably be merged into the domain initializer

Reply:
That's no longer `minimal`. I think it's clear that after this PR the initialization will still need some work to get it in better shape. This sort of change should be part of a **series** of PRs that do that.

Reply:
sure 👍